crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
```

Comment:
Avoid blocking the main thread with `Thread.sleep`; consider using asynchronous processing.
```suggestion
// Inefficient way to calculate the sum of elements
    var sum = 0
    fun calculateSum() {
        sum = largeList.sum()
        // typo in println() function
        println("Sum: $sum")
        // Unnecessary object creation inside a loop
        val tempObject = Any() // Create the object once outside the loop
        for (i in 0 until 100000) {
            // Use tempObject if needed
        }
        // Inefficient string concatenation
        var longString = ""
```